File: DataInfrastructure/Entities/BookEM.cs
using System;
using System.Collections.Generic;

namespace DataInfrastructure.Entities
{
    public class BookEM
    {
        public long? Id { get; set; }
        public string Name { get; set; }
        public int Rate { get; set; }
        public int Pages { get; set; }
        public List<AuthorEM> Authors { get; set; }
        public DateTime CreatedDate { get; set; }

        public BookEM()
        {
            Authors = new List<AuthorEM>();
        }
    }
}

File: EntityModelsVB/BookEM.vb
Public Class BookEM
    Public Property Id As Long?
    Public Property Name As String
    Public Property Rate As Integer
    Public Property Pages As Integer
    Public Property CreatedDate As DateTime
    Public Property Authors As List(Of AuthorEM)

    Public Sub New()
        Authors = New List(Of AuthorEM)()
    End Sub
End Class
